templates/docs/examples/patterns/password-toggle/default.html

```diff
@@ -0,0 +1,50 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Password toggle field{% endblock %}
+
+{% block standalone_css %}patterns_password-toggle{% endblock %}
+
+{% block content %}
+<div class="p-password-toggle">
+  <div class="p-password-toggle__container">
+    <label for="password">Password</label>
+    <button class="p-password-toggle__button" aria-controls="password">
+      <i class="p-icon--show"></i>
+      <span class="p-password-toggle__label">
+        Show
+      </span>
+    </button>
+  </div>
+  <input type="password" name="password" id="password" value="letmein">
+</div>
+
+<script>
```

Comment:
We try to keep JS separate from HTML examples, see the [tabs example](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/examples/patterns/tabs/content.html#L54).